client/src/pages/ChefProfile.js

```diff
@@ -224,18 +267,23 @@ function ChefProfile(props) {
                         direction="column"
                         justify="center"
                         alignItems="center"
-                        className={classes.color1}
                     >
                         <Grid item>
                             <Typography variant="h5">
-                                <Box mt={5} fontWeight="fontWeightBold">
-                                    {`${user.profile.firstName}'s`} Menu:
-                                </Box>
+                                {chefInfo.chefProfile._id !== currentChef._id ? (
+                                    <Box mt={5} fontWeight="fontWeightBold">
+                                        {`${chefInfo.firstName}'s`} Menu:
+                                    </Box>
```

Comment:
Since the `Box` part is the same, we can do the conditional for just the text:
```
<Box mt={5} fontWeight="fontWeightBold">
  {chefInfo.chefProfile._id !== currentChef._id ? `${chefInfo.firstName}'s` : `Your`} Menu:
</Box>
```